packages/react-components/src/button/src/Button.css

```diff
@@ -83,82 +84,95 @@ a.o-ui-button {
 /* SOLID | STATE | ACTIVE */
 .o-ui-button-solid:active,
 .o-ui-button-solid.o-ui-button-active {
-    background-color: var(--marine-900);
+    background-color: var(--foreground-color-5-active);
 }
 
 /* SOLID | STATE | DISABLED | ACTIVE */
 .o-ui-button-solid:active[disabled],
 .o-ui-button-solid.o-ui-button-active[disabled],
 .o-ui-button-solid:active.o-ui-button-disabled,
 .o-ui-button-solid.o-ui-button-active:disabled {
-    background-color: var(--marine-500);
+    background-color: var(--foreground-color-5);
 }
 
 /* SOLID | PRIMARY */
 .o-ui-button-solid.o-ui-button-primary {
-    background-color: var(--primary-500);
+    background-color: var(--foreground-color-brand);
```

Comment:
Again I would have expected `--foreground-color-primary`. Maybe it's me who doesn't understand the system well.